.github/workflows/lint-python.yml

```diff
@@ -6,16 +6,19 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
-      - run: pip install build setuptools wheel
-      - run: pip install bandit black codespell flake8 isort mypy pytest pyupgrade safety
-      - run: bandit --recursive --skip B101,B105,B106,B110,B303,B324,B404,B603 .
+      - run: pip install -r requirements_build.txt
+      - run: bandit -c pyproject.toml -r .
       - run: black --check . || true
-      - run: codespell || true  # --ignore-words-list="" --skip=""
+      - run: codespell || true
+      # flake8 doesn't currently support pyproject.toml config - there is a wrapper pflake8 which does
       - run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
       - run: flake8 . --count --exit-zero --max-complexity=29 --max-line-length=167 --show-source --statistics
-      - run: isort --check-only --profile black .
-      - run: pip install -e . # editable mode needs build backend support for PEP660
-      - run: mypy --ignore-missing-imports . || true
+      - run: isort -c .
```

Comment:
When typing on the command line it will usually be preferred to use single-letter options but when writing in scripts and similar files, it is preferred to use longer names to make things more self-documenting and less cryptic.

Reply:
Done